Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_gg.gnut

```diff
@@ -64,6 +64,10 @@ void function OnPlayerKilled( entity victim, entity attacker, var damageInfo )
 	}
 	else
 	{
+		// Heal & stim attacker after kill
```

Comment:
```suggestion
		// Heal attacker after kill
```
we dont stim them, missed this on my first review

Reply:
"how could you tell I copy pasted the stupid gun game code"